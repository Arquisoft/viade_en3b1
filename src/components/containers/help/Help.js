import React from 'react';
import FaqQuestions from './FaqQuestions';
import NavBar from '../../ui/main/NavBar';
import Footer from '../../ui/main/Footer';
import { Container, makeStyles } from '@material-ui/core';
import svgIcon from '../../../assets/img/icons/HelpSupport.svg';

export default function Help() {

    const classes = useStyles();

    return (
        <div>
            <NavBar />
            <Container
                component="main"
                className={classes.main}
                maxWidth="md"
            >
                <img alt="Help Support" src={svgIcon} />
                <FaqQuestions />
            </Container>
            <Footer />
        </div>
    )
}

const useStyles = makeStyles((theme) => ({
    main: {
        marginTop: theme.spacing(3),
        marginBottom: theme.spacing(8),
    },
}));