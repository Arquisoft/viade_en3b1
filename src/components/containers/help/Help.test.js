import Help from "./Help";

test("Help", () => {
    expect(Help).toBeDefined();
});

test("Renders Correct", () => {
    const wrapper = render(<Help />);
    expect(wrapper).toMatchSnapshot();
});