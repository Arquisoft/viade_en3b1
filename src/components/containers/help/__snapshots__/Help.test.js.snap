// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Correct 1`] = `
<div>
  <div
    class="makeStyles-root-2"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-3 MuiToolbar-gutters"
      >
        <div
          class="MuiGrid-root makeStyles-link-5"
        >
          <div
            class="MuiGrid-root MuiGrid-container"
          >
            <a
              class="MuiTypography-root MuiLink-root MuiLink-underlineNone MuiTypography-colorInherit"
              href="#/home"
            >
              <div
                class="MuiGrid-root MuiGrid-container"
              >
                <div
                  class="MuiGrid-root MuiGrid-item"
                >
                  <img
                    alt="Viade Logo"
                    src="logo_2.svg"
                    style="height:2.5rem;width:2.5rem"
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item"
                >
                  <h4
                    class="MuiTypography-root MuiTypography-h4"
                  >
                    ViaDe
                  </h4>
                </div>
              </div>
            </a>
          </div>
        </div>
      </div>
    </header>
  </div>
  <main
    class="MuiContainer-root makeStyles-main-1 MuiContainer-maxWidthMd"
  >
    <img
      alt="Help Support"
      src="HelpSupport.svg"
    />
    <div>
      <div
        class="MuiBox-root MuiBox-root-197"
      >
        Frequently asked questions (FAQ)
      </div>
      <div>
        <div
          style="margin-top:2rem"
        >
          <div
            class="MuiBox-root MuiBox-root-198"
          >
            What is Solid?
          </div>
          <div
            class="MuiBox-root MuiBox-root-199"
          >
            It is an open source project, created by Tim Berners-Lee, that aims to achieve a complete, 
            decentralized web allows users to decide where to store their own data.
          </div>
        </div>
        <div
          style="margin-top:2rem"
        >
          <div
            class="MuiBox-root MuiBox-root-200"
          >
            What is a Pod?
          </div>
          <div
            class="MuiBox-root MuiBox-root-201"
          >
            A Pod is where data is stored on the Web with Solid. (see quote). A user may store their data in one Pod or several Pods, 
            and applications read and write data into the Pod depending on the authorisations granted by the user or users associated to that Pod.
          </div>
        </div>
        <div
          style="margin-top:2rem"
        >
          <div
            class="MuiBox-root MuiBox-root-202"
          >
            Where are my routes saved?
          </div>
          <div
            class="MuiBox-root MuiBox-root-203"
          >
            All your routes, as well as the media files associated with them are stored in your own Pod. When you disconnect from our app,
            no data is stored on our servers.
          </div>
        </div>
        <div
          style="margin-top:2rem"
        >
          <div
            class="MuiBox-root MuiBox-root-204"
          >
            Can I store any number of routes in my Pod?
          </div>
          <div
            class="MuiBox-root MuiBox-root-205"
          >
            Yes, you can store any number of routes that you want in your Pod. To store a route, you can do it manually in
            your Pod, or you can use our app to create or even import some.
          </div>
        </div>
        <div
          style="margin-top:2rem"
        >
          <div
            class="MuiBox-root MuiBox-root-206"
          >
            My routes doesn't seem to load. Why is that?
          </div>
          <div
            class="MuiBox-root MuiBox-root-207"
          >
            If you have a big number of routes associated to your account, you may experience load times bigger than usual. 
            Don't worry about that, if you wait enough, you will be able to see all your routes.
          </div>
        </div>
        <div
          style="margin-top:2rem"
        >
          <div
            class="MuiBox-root MuiBox-root-208"
          >
            I can't see all my routes listed. What is the reason?
          </div>
          <div
            class="MuiBox-root MuiBox-root-209"
          >
            ViaDeEn3B1's app only works with routes that comply with ViadeSpec's standard. This means that the application can only list 
            JSON-LD routes that are correctly built.
          </div>
        </div>
        <div
          style="margin-top:2rem"
        >
          <div
            class="MuiBox-root MuiBox-root-210"
          >
            When I try to import a route, I get an error. Why is that?
          </div>
          <div
            class="MuiBox-root MuiBox-root-211"
          >
            You can only import GPX, JSON and JSON-LD routes. Those files may contain some errors, so they get rejected and ignored
            when trying to import them.
          </div>
        </div>
        <div
          style="margin-top:2rem"
        >
          <div
            class="MuiBox-root MuiBox-root-212"
          >
            I can't import my GPX route. Why?
          </div>
          <div
            class="MuiBox-root MuiBox-root-213"
          >
            Our GPX-import system is optimized specially for achieving interoperability with Strava's routes. This doesn't mean that
            you can only import files that come from Strava, but you may encounter some problems if your GPX file has an strange internal structure.
          </div>
        </div>
      </div>
    </div>
  </main>
  <div>
    <footer
      class="makeStyles-footer-216"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthXs"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-5"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              ViaDe En3B1
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 MuiTypography-colorTextSecondary"
            >
              Made by 
              <a
                class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                href="https://www.linkedin.com/in/pablocanalsuarez/"
                style="color:#302c58"
                target="_blank"
              >
                Pablo Ca√±al
              </a>
            </h6>
            <div>
              <a
                class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                href="https://www.linkedin.com/in/pablocanalsuarez/"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                  />
                </svg>
              </a>
              <a
                class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                href="mailto:uo264903@uniovi.es"
                style="margin-left:0.5rem"
                target="_blank"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4l-8 5-8-5V6l8 5 8-5v2z"
                  />
                </svg>
              </a>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-flexItem MuiDivider-vertical"
          />
          <div
            class="MuiGrid-root MuiGrid-item"
            style="margin-top:1rem"
          >
            <a
              class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
              href="https://github.com/Arquisoft/viade_en3b1/"
              target="_blank"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                />
              </svg>
            </a>
          </div>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
